### PR TITLE
Clean up many pep8 warnings

### PR DESCRIPTION
**Pull request by anguslees**
Some errors, most whitespace changes.

### CODE REVIEW

File: craton/inventory/db/sqlalchemy/api.py [hunk 1]
```diff
@@ -44,16 +38,6 @@ def get_backend():
     return sys.modules[__name__]
 
 
-def require_admin_context(f):
```
Comment (anguslees) [inline craton/inventory/db/sqlalchemy/api.py]: `is_admin_context` is undefined, and this function is unused (afaict).

File: craton/inventory/api/middleware.py [hunk 2]
```diff
@@ -86,7 +88,7 @@ def process_request(self, request):
         headers = request.headers
 
         try:
-            if headers["X-Identity-Status"] is "Invalid":
+            if headers["X-Identity-Status"] == "Invalid":
```
Comment (jimbaker) [inline craton/inventory/api/middleware.py]: Good catch, just the sort of thing that works well in testing

**Comment by jimbaker** [comment]
From what I can tell, Sulo applied more or less the same PEP 8 fixes to master (which is currently broken). But we still need the fix for `craton/inventory/api/middleware.py` re the buggy usage of `is` vs `==`.

File: craton/inventory/api/middleware.py (hunk 2 above)
Comment (sulochan) [inline craton/inventory/api/middleware.py]: +1

**Comment by sulochan** [comment]
yeah most of this is fixed in my last commit but i didn't touch hosts.py  and other fils with pep8 error etc that was not relevant on that patch.

File: craton/inventory/db/sqlalchemy/api.py (hunk 1 above)
Comment (sulochan) [inline craton/inventory/db/sqlalchemy/api.py]: this is fixed in the sense that function exists now but i am passing it until we get the users table (and context) properly sorted.

**Comment by jimbaker** [comment]
I'm going to short circuit this review process by rejecting this PR - we are not going to be fixing up this branch at this point, given merge conflicts and multiple people working this problem.

Since my Travis CI branch also fixed PEP 8 and Python 3.4 support (since it surfaced the latter), I figured it might as well include the `is` vs `==` bug fix so we didn't lose that small but important fix that Gus made.

**Pull request closed without merging**